docs/HOWTO.md

```diff
@@ -1,45 +1,34 @@
-How to create and track horizontal issues
-=========================================
+For Working Groups
+==================
 
-# What is a horizontal issue?
+* Apply the *-tracker label in your own repository to draw a horiztontal review group's attention to an issue in one of your own repositories.  Horizontal review groups may also apply the label if they are interested in tracking a particular issue.
 
-Horizontal issues are issues on specifications raised by an [horizontal group](https://www.w3.org/wiki/DocumentReview#How_to_get_horizontal_review).
+* Horizontal review groups may apply the *-needs-resolution to issues they expect to be resolved before the specification moves to a new maturity level.  
 
-Those issues are tracked more closely due to their potential wide impact on users, organizations or the Web architecture.
+* If *-needs-resolution is applied to an issue, automatic tooling will remove the *-tracker label.
 
-# Horizontal issues and W3C Process
+## Do these labels replace the need to ask for horizontal review?
 
-Horizontal issues should be raised and tracked as early possible. Some get raised before a specification is adopted by a W3C Working Group to be on the Standard track.
+No.  The is *-tracker label asks for input on a specific issue.  For through review, follow [the published process for requesting horitzontal review](https://www.w3.org/wiki/DocumentReview#How_to_get_horizontal_review)
```

Comment:
Before scheduling a transition request, the Working Group should also ensure that all issues labelled *-needs-resolution have received a reply from the horizontal group indicating that they are satisfied or object to the resolution arrived at.

Reply:
I like this add

Reply:
on second thought... there's got to be a way to say this better, encouraging the reaching of consensus.  How's this?

Before making a transition request, Working Groups should address all issues labelled *-needs-resolution, attempting to reach agreement with the reviewer on an appropriate resolution.